plugin-build/modulegraph/src/main/kotlin/dev/iurysouza/modulegraph/gradle/CreateModuleGraphTask.kt

```diff
@@ -85,21 +111,21 @@ abstract class CreateModuleGraphTask : DefaultTask() {
     @TaskAction
     fun execute() {
         runCatching {
-            val graphOptions = GraphOptions(
-                theme = theme.getOrElse(Theme.NEUTRAL),
-                orientation = orientation.getOrElse(Orientation.LEFT_TO_RIGHT),
-                focusedNodesRegex = focusedModulesRegex.orNull?.let { Regex(it) },
-                showFullPath = showFullPath.getOrElse(false),
-                linkText = linkText.getOrElse(LinkText.NONE),
-                setStyleByModuleType = setStyleByModuleType.getOrElse(false),
-            )
-            val mermaidGraph = Mermaid.generateGraph(graphModel.get(), graphOptions)
-            ReadmeWriter.appendOrOverwriteGraph(
-                mermaidGraph = mermaidGraph,
-                readMeSection = heading.get(),
-                readmeFile = outputFile.get().asFile,
-                logger = logger,
-            )
+            val results =
+                graphModels.orNull ?: error("Graph models have not been computed on time!")
```

Comment:
Hey! Im curious about this one.

Have you experienced that? It seems like a cryptic error for library users. Judging by the message, it feels like it shouldn't be possible.
What are your thoughts on this?

Reply:
You are right, I don't think this should ever be possible. I've updated the error message to make this clear, and if it ever does happen then we have an obvious message for users to reference during bug reports